A4G/donations.yml

```diff
@@ -5,6 +5,24 @@
 #                monthly to a charity you will have to add new entries here on
 #                a monthly basis.
 
+- ticker: A4G  
+
+  recipient: 
+    name: Save the Children
+    address: 30 Orange Street
+    tel: +44 (0)20 3272 0300
+    contact: info@savethechildren.org
+
+  ada: 164.9
+  usd: 219
+  timestamp: 1627902000   
+
+  files:
+    - A4G/2021-08-01-transaction-confirmation.jpg
+    - A4G/2021-08-01-payment-confirmation.jpg
```

Comment:
also, the file names are dated to the 2nd of august.

Reply:
ok changed the file names